src/main/resources/content/EOI/eoiType.json

```diff
@@ -1,46 +1,52 @@
 {
```

Comment:
is it possible to keep the folder eoi in lowercase? In current develop is in lowercase.